charts/frr-k8s/README.md

```diff
@@ -93,6 +93,7 @@ Kubernetes: `>= 1.19.0-0`
 | prometheus.serviceMonitor.metricRelabelings | list | `[]` |  |
 | prometheus.serviceMonitor.relabelings | list | `[]` |  |
 | prometheus.serviceMonitor.tlsConfig.insecureSkipVerify | bool | `true` |  |
+| prometheus.whmetricsPort | int | `7672` |  |
```

Comment:
Do we need this port because now we open a hostport so the other value is taken?

Reply:
yes